paywall/src/unlock.js/CheckoutUIHandler.ts

```diff
@@ -1,6 +1,24 @@
 import IframeHandler from './IframeHandler'
 import { PostMessages } from '../messageTypes'
-import { PaywallConfig } from '../unlockTypes'
+import { PaywallConfig, Balance } from '../unlockTypes'
+
+// Arbitrarily defined number of tokens to assume a managed user
+// account holds for any given stablecoin
+export const defaultBalance = '35'
+
+export const injectDefaultBalance = (oldBalance: Balance): Balance => {
+  const newBalance: Balance = {}
+  const tokens = Object.keys(oldBalance)
+  tokens.forEach(token => {
+    if (token.startsWith('0x')) {
```

Comment:
My understanding is that balances of ERC20 tokens are all stored with the currency contract address as the key. Is that assumption useful?